charts/prometheus-mysql-exporter/templates/servicemonitor.yaml

```diff
@@ -30,6 +30,38 @@ spec:
 {{ toYaml . | trim | indent 4 -}}
   {{- end }}
   endpoints:
+  {{- if .Values.serviceMonitor.multipleTarget }}
+  {{- range  .Values.serviceMonitor.targets }}
+    - path: /probe
```

Comment:
is that /metrics or /probe?

Reply:
It's `/probe` when you enable multitarget: https://github.com/prometheus/mysqld_exporter#multi-target-support